reference/resources/payment_service_definition.yaml

```
@@ -95,6 +95,24 @@ properties:
       - GB
       - DE
 
+  supported_features:
+    type: object
+    description: |-
+      Features supported by the payment definition.
+    properties:
+      payment_tokenization:
+        type: boolean
+        example: true
+        description: Supports storing a payment method via tokenization.
+      three_d_secure_hosted:
+        type: boolean
+        example: true
+        description: Supports hosted 3D secure with a redirect.
+      three_d_secure_pass_through: 
+        type: boolean
+        example: false
+        description: Supports passing 3D secure data to the underlying processor.
```

```suggestion
        description: Supports passing 3-D Secure data to the underlying processor.
```

I think this is the official spelling.